mantidimaging/core/io/test/utility_test.py

```diff
@@ -1,20 +1,21 @@
 # Copyright (C) 2022 ISIS Rutherford Appleton Laboratory UKRI
 # SPDX - License - Identifier: GPL-3.0-or-later
 
-import os
 from pathlib import Path
 
-from mantidimaging.helper import initialise_logging
 from mantidimaging.core.io import utility
-from mantidimaging.test_helpers import FileOutputtingTestCase
+from pyfakefs.fake_filesystem_unittest import TestCase
 
 
-class UtilityTest(FileOutputtingTestCase):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class UtilityTest(TestCase):
+    def setUp(self) -> None:
+        self.setUpPyfakefs()
+        # import pyfakefs
+        # self.fs.os = pyfakefs.fake_filesystem.OSType.WINDOWS  # Allow testing with a different OS style
```

Comment:
As this may be forgotten about in the future, we should probably include the removal of this as a requirement/acceptance criteria for the final issue related to this work and maybe include it as a code snippet in our documentation somewhere for developers.